Sources/BrowserServicesKit/Autofill/AutofillUserScript.swift

```diff
@@ -19,16 +19,53 @@
 
 import WebKit
 
-public class AutofillUserScript: NSObject, UserScript {
+public protocol AutofillUserScriptDelegate: AnyObject {
+    func setSize(height: CGFloat, width: CGFloat)
+}
+public protocol ChildAutofillUserScriptDelegate: AnyObject {
+    func clickTriggered(clickPoint: NSPoint)
```

Comment:
I don't see any use case in BSK for this - can this protocol be removed?

Reply:
No, It's used in the MacOS patch.

Reply:
So why not put it there instead? Is there any hint in BSK how this protocol is supposed to be used in the platform code?